integration/smartbft/smartbft_block_deliverer_test.go

```diff
@@ -0,0 +1,205 @@
+/*
+Copyright IBM Corp All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package smartbft
+
+import (
+	"context"
+	"encoding/binary"
+	"fmt"
+	docker "github.com/fsouza/go-dockerclient"
+	"github.com/hyperledger/fabric/common/ledger/blkstorage"
+	"github.com/hyperledger/fabric/common/ledger/blockledger/fileledger"
+	"github.com/hyperledger/fabric/common/metrics/disabled"
+	"github.com/hyperledger/fabric/integration/nwo/commands"
+	"github.com/hyperledger/fabric/internal/pkg/comm"
+	ginkgomon "github.com/tedsuo/ifrit/ginkgomon_v2"
+	"github.com/tedsuo/ifrit/grouper"
+	"os"
+	"path/filepath"
+	"syscall"
+	"time"
+
+	"github.com/hyperledger/fabric-protos-go/common"
+	ordererProtos "github.com/hyperledger/fabric-protos-go/orderer"
+	"github.com/hyperledger/fabric/integration/nwo"
+	"github.com/hyperledger/fabric/integration/ordererclient"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gbytes"
+	"github.com/tedsuo/ifrit"
+)
+
+func smartBftBlockDelivererTest(
+	testDir string,
+	client *docker.Client,
+	network *nwo.Network,
+	networkProcess ifrit.Process,
+	ordererProcesses []ifrit.Process,
+	peerProcesses ifrit.Process) {
+
+	networkProcess = nil
+	ordererProcesses = nil
+	peerProcesses = nil
+	var err error
+
+	/* Create a client*/
+	client, err = docker.NewClientFromEnv()
+
+	Expect(err).NotTo(HaveOccurred())
+
+	/* Create networkConfig for multiple orderers */
+	networkConfig := nwo.MultiNodeSmartBFT()
+	networkConfig.Channels = nil
+
+	channel := "testchannel1"
+
+	fmt.Println(client.Endpoint())
```

Comment:
Please don't use fmt.Println in order to track the progress of the test. Use `By(fmt.Sprintf(""))` etc

Reply:
Done